src/index.ts

```diff
@@ -100,6 +103,9 @@ export default function factory(): CompilerPlugin {
             const codeStyle = new CodeStyle(context);
             program.plugins.add(codeStyle);
 
+            const codePerformance = new CodePerformance(context);
+            program.plugins.add(codePerformance);
+
```

Comment:
I'd prefer not to have yet another plugin, so let's move the performance checking functionality into the `codeStyle` plugin. That should also reduce a bunch of the boilerplate code from this PR.

Reply:
Changes were moved to code style plugin